lib/widgets/icons.dart

```diff
@@ -57,14 +57,17 @@ abstract final class ZulipIcons {
   /// The Zulip custom icon "read_receipts".
   static const IconData read_receipts = IconData(0xf10b, fontFamily: "Zulip Icons");
 
+  /// The Zulip custom icon "star".
+  static const IconData star = IconData(0xf10c, fontFamily: "Zulip Icons");
```

Comment:
Oh, one other comment: let's call this icon `star_filled`. That way it matches the name Vlad gave it in the mockup, and it makes the name more descriptive since we may have an unfilled star in the future.